### PR TITLE
Feature/new create nations ui

### PR DESCRIPTION
**Pull request by seland**
#### Changes
Create nation screen UI
Save/delete/submit draft functionality

### CODE REVIEW

File: src/global/Screens.js [hunk 1]
```diff
@@ -74,13 +88,13 @@ const Screens = {
     screen: 'Pangea.NationsScreen',
     label: i18n.t('screens.nations.tabTitle'),
     icon: AssetsImages.TabIcons.nations,
-    title: i18n.t('screens.nations.title'),
+    title: '',
```
Comment (florianlenz) [inline src/global/Screens.js]: Is this correct? Just asking since it's an title which is an empty string.

File: src/global/Screens.js [hunk 2]
```diff
@@ -74,13 +88,13 @@ const Screens = {
...
     navigatorStyle,
   },
   NATION_DETAILS_SCREEN: {
     screen: 'Pangea.NationDetailsScreen',
-    title: i18n.t('screens.nationDetails.title'),
-    navigatorStyle,
+    title: '',
```
Comment (florianlenz) [inline src/global/Screens.js]: Is this correct? Just asking since it's an title which is an empty string.

File: src/global/Styles.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { MediaQueryStyleSheet } from 'react-native-responsive'
 import Colors from './colors'
-import { normalizer} from '../utils/normalizer'
+import { normalizer, normalWidthMargin } from '../utils/normalizer'
```
Comment (florianlenz) [inline src/global/Styles.js]: Normalizer of what exactly? Can you change the naming here?

File: src/global/Screens.js (hunk 1 above)
Comment (seland) [reply src/global/Screens.js]: Yes, that screens don't have a title (it's shown as large title in a custom way).

File: src/global/Screens.js (hunk 2 above)
Comment (seland) [reply src/global/Screens.js]: Yes, that screens don't have a title (it's shown as large title in a custom way).

File: src/global/Styles.js (hunk 3 above)
Comment (seland) [reply src/global/Styles.js]: Renamed.

**Pull request merged**